src/tests/suite/daos_nvme_recovery.c

```diff
@@ -115,7 +115,7 @@ nvme_recov_1(void **state)
 	 *	  is ready.
 	 */
 	print_message("Waiting for faulty reaction done...\n");
-	sleep(60);
+	sleep(180);
```

Comment:
Do we really need to wait that long? Some value larger than the monitoring check interval (60) like 70 isn't enough?

Reply:
I tried with 120 (2 of the monitoring check time cycles -> NVME_MONITOR_PERIOD=60) and it still wasn't enough. I could maybe reduce to 150, however thought 3 cycles would be good measure.

Reply:
honestly this is very inconsistent behavior - I tried testing just now with smaller wait durations, and sometimes the state will progress to OUT and sometimes it will remain at TEARDOWN. seems like more investigation is needed into what is blocking the poller occasionally

Reply:
yup, I think you could collect more debug log to see why the state wasn't transited within 2 monitor cycles.